### ISSUE TITLE
Accessibility: use descriptive, action-oriented wording for in-line links

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-registration#3098]
#### Describe the task

Have descriptive link text for inline hyperlinks to assist screen readers in conveying a clear purpose and destination for the link. Right now links are not always descriptive and can lack context.

#### Acceptance Criteria

**Successful new registration submission screen:**
 
**Change:** 
To report any of these events, please use the following link: [Link to form](https://submit.digital.gov.bc.ca/app/form/submit?f=d26fb011-2846-44ed-9f5c-26e2756a758f)
Staff will review the information provided and administer changes in BCIERS as needed. For questions reach out to [GHGRegulator@gov.bc.ca](mailto:GHGRegulator@gov.bc.ca).
 
TO
 
“Report an event and staff will review the information provided and administer changes in BCIERS as needed. For questions reach out to ghgregulator@gov.bc.ca.”
 
**Submit a report**
2nd screen - Person responsible
 
**Change**
See that information is incorrect or need to add a new contact? Update it here in [Contacts.](https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/contacts)

TO
 
[Add or edit contact information](url).
 
 
Submit a report
3rd screen - Review facilities
Linear Facilities Operations must register and report for all large and medium facilities, as well as a small aggregate, if applicable. Don’t see a facility? [Add it here](https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/operations/954c0382-ff61-4e87-a8a0-873586534b54/facilities/add-facility) and then click on the ‘Sync latest data from Administration’ button to update this list of facilities.
 
TO
 
Add a new facility and click the sync button to update the list of facilities to select from if needed.
 
Linear Facilities Operations must register and report for all large and medium facilities, as well as a small aggregate, if applicable.

#### Definition of Done

- [ ] first
- [ ] second
- [ ] third

#### Additional context

- Add any other context about the task here.
- Or here
